changelogs/fragments/13-fix-sanity-tests.yaml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - digital_ocean - fixing sanity tests (https://github.com/ansible-collections/community.digitalocean/issues/13).
```

Comment:
Changing documentation is not really noteworthy. What you should document in the changelog are user-facing changes, i.e. any modification you made to `argument_spec`.